src/work-allocation-2/components/find-person/find-person.component.ts

```diff
@@ -59,7 +61,7 @@ export class FindPersonComponent implements OnInit {
   }
 
   public updatedVal(currentValue) {
-    if (currentValue && currentValue.length > this.minSearchCharacters) {
+    if (currentValue && currentValue.length >= this.minSearchCharacters) {
```

Comment:
Same as above

Reply:
I now set the this.minSearchCharacters=3, that make it more sense for minSearchCharacters, this code will keep as >=